content/en/apps/guides/offline/_index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Offline CHT Server"
+linkTitle: "Offline CHT"
+weight: 100
+description: >
+ Running an instance of CHT Core server with no internet connection
+---
+
+## Introduction
+
+The CHT is built as an offline first application. This applies to clients, either  browsers or Android applications, connecting to the CHT server.  The server itself assumes it has Internet connectivity to provide services such as DNS, software updates and general use connectivity.  This document explores what it looks like when the CHT server is offline without these services available.
+
+Running a CHT server offline requires no modifications to the CHT itself.  Instead, supporting services normally found online are replicated locally. 
+
+## Considerations
+
+An offline CHT server is most appropriate for a development environment.  There are serious implications to consider before deploying an offline server in a production environment which are most often ignored for development.  Offline solution should have a plan in place for:
```

Comment:
What do you think about using stronger wording around the development environment here and avoiding the use of production altogether?

 "Here are the instructions for setting this up in a development environment. This has not been deployed in a real environment, due to:"

Reply:
Hmmm, I think this is a very strongly worded sentence already:

>  There are serious implications to consider before deploying an offline server in a production environment

Further, saying:

> Here are the instructions for setting this up in a development environment

At this spot in the docs doesn't really make sense, as we cover none of considerations for a development environment.

Maybe we could change the prior paragraph from:

> Running a CHT server offline requires no modifications to the CHT itself. Instead, supporting services normally found online are replicated locally.

To have more of what you were looking for like this?

> Running a CHT server offline requires no modifications to the CHT itself. Instead, supporting services normally found online are replicated locally.
> 
> **Note:** Due to the considerations below, a production offline CHT server has not been deployed in a real environment.

However, I kinda think this is maybe _too_ strongly worded?  I dunno...thoughts?  My inclination is to leave it as is given we dedicate about 1/3 of the total content saying why this might not be a good idea (per all the considerations).

Reply:
I want to stay away from the word production altogether unless we are committed to providing long-term support when these deployments contain real data. If that's a deployment scenario we want to promote in production, than we need to meet the column titles in our [Product Supported Systems](https://docs.google.com/spreadsheets/u/1/d/1uo-8D44YNSHibP5y_5HObJQPE1r4jBpOMiZaWbKbK4c/edit#gid=0)